vnext/proposals/active/mouse-reconcile-desktop.md

```diff
@@ -0,0 +1,107 @@
+- Ensuring mouse behavior is consistent across the two platforms we support
+- Once our two desktop platforms are aligned with React Native Web, we upstream this implementation to core [issue TBD]()
+
+## `hovered`
+Property on `Pressable` that determines whether or not the mouse is currently within the bounds of the designated content.
+
+| TYPE | REQUIRED | PLATFORM |
+|:--:|:--|:--|
+| boolean | Yes | Windows, macOS |
+
+[React Native Web](https://github.com/necolas/react-native-web/commit/1afd9de0455965a852bad42ad2ffab957f7ffca0#diff-9fe3ab194c72e9837a8ffc3e7c44f33d) introduced  `hovered` on `Pressable` to enable common mouse scenarios. In order to align and make this complete the work is as follows:
+
+- Windows has added `hovered` and marked `onMouseEnter/Leave` as to be deprecated [issue TBD]()
+- Add `hovered` to `Pressable` [issue TBD]()
+- macOS has added 'hovered' and marked their View component implementation of `onMouseEnter/Leave` as deprecated [issue TBD]()
+
+### `hovered` example
+
+``` js
+<Pressable style={({ hovered, pressed, focused }) => {
+          console.log(focused);
+          let backgroundColor = 'white';
+          if (hovered) {
+            backgroundColor = 'lightgray';
+          }
+          if (pressed) {
+            backgroundColor = 'lightblue';
+          }
+          return {
+            padding: 10,
+            margin: 10,
+            borderWidth: 1,
+            borderColor: focused ? 'red' : null,
+            backgroundColor,
+            outlineWidth: 0
+          };
+        }}
+      >
+      <Text>Nested pressables</Text>
+    </Pressable>
+  </Pressable>
+</View>
+```
+
+## Outstanding issues
+There are some outstanding issues that need to be address specifically for desktop scenarios when it comes to fully implementing mouse hover and leave behavior. They are as follows:
+
+- Prevent mouse input from being 'eaten' in ScrollView scenarios [issue #5867](https://github.com/microsoft/react-native-windows/issues/5867)
+- Mouse leave event not firing correctly in Scrollable FlatList scenarios [issue #4614](https://github.com/microsoft/react-native-windows/issues/4614)
+- The hover events/indication for overlapping content prioritizes items "closer" in z-order [issue #5901](https://github.com/microsoft/react-native-windows/issues/5901)
+- Remove the tapped event handler on the `View` component [issue #2349](https://github.com/microsoft/react-native-windows/issues/2349)
+
+## onPointerXX callbacks
+The following callbacks are available on `View` component (and get passed through to Pressable) in Windows and macOS to cover the most common use cases where mouse handling would occur.
+
+The below will need to be implemented for macOS as well [issue TBD]()
+
+| API | Args | Returns | Description |
+|:---:|:----:|:-------:|----|
+| onPointerOver | IPointerEvent | void | Fires when a pointing device is moved within the hit test boundaries of a element. |
```

Comment:
Why the choice of onPointerOver/onPointerLeave? The pairs, per the [web ](https://www.w3.org/TR/pointerevents2/#the-pointerenter-event)spec, are:  onPointerOver/onPointerOut and onPointerEnter/onPointerLeave.

onPointerOver/onPointerOut are a little weird on web and I'm wondering what the desired implementation is (i.e. should it fully match?). The brief definitions here make it sound like they simply track the top-most element under the cursor. That is sort-of true, but in web at least they also bubble. That means that if you have a page that looks like:
```
<div id="page>
    <div id="button>
        <div id="text">CONTENT</div>
    </div>
</div>
```
and you mouse over button and then into the text, you'll get the following event sequence:
1. `pointerover button`
2. `pointerout button`
3. `pointerover text`
4. `pointerover button` (bubbled from text)

onPointerEnter/onPointerLeave match with the current touch algorithm (called out in #4274 and #5901). Their sequence for the same action would be:
1. `pointerenter button`
2. `pointerenter text`

FWIW, I think onPointerEnter/onPointerLeave are more useful. As you can see above, they can be used to create a button that with a hover state, and have that hover state remain active when the mouse enters child elements.

Reply:
I was just attempting to match the system on web, since that was my frame of reference here. But if we can shorten the event/bubble tree by removing onPointerOver I'm good with that. I just want to make sure all the necessary events are available for devs to listen to in order to enable basic and high-pri mouse scenarios.

Reply:
Gotcha. I can only speak for myself and what would be useful to Garrison. I imagine there are use cases for onPointerOver, so it'd be good to hear from other app devs what their thoughts are.